cmake/platform.cmake

```diff
@@ -41,17 +41,9 @@ include(cmake/platform/toolcfg/${OMR_TOOLCONFIG}.cmake OPTIONAL)
 # Verify toolconfig!
 include(cmake/platform/toolcfg/verify.cmake)
 
+# Include useful tool macros.
 include(cmake/AddPrefix.cmake)
-
-# Remove a specified option from a variable
-macro(omr_remove_option var opt)
-   string( REGEX REPLACE
-      "(^| )${opt}($| )"
-      ""
-      ${var}
-      "${${var}}"
-      )
-endmacro(omr_remove_option)
+include(cmake/RemoveOption.cmake)
```

Comment:
Moving this off into its own file seems a bit overkill. Maybe a `utilities.cmake` or something similar to collect small tools ?